### PR TITLE
add getDevice(s) getChannel(s) tests

### PR DESCRIPTION
**Pull request by manuIO**

### CODE REVIEW

**Comment by drasko** [comment]
LGTM

File: api/message.go [hunk 1]
```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/go-zoo/bone"
 )
 
+var errSenML error
```
Comment (drasko) [inline api/message.go]: I do not like the idea of having global var. Rather make function `writeMessage` to return an error.

Comment (manuIO) [inline api/message.go]: I don't like either, I'll change it. I did it like that to don't change nats handler here: https://github.com/mainflux/mainflux-core/blob/master/api/nats.go#L49

**Comment by drasko** [comment]
Merged, thanks!

**Pull request merged**